lib/scenic/component.ex

```diff
@@ -66,8 +66,44 @@ defmodule Scenic.Component do
 
   alias Scenic.Primitive
 
-  @callback add_to_graph(map, any, list) :: map
-  @callback verify(any) :: any
+  @optional_callbacks add_to_graph: 3, info: 1
+
+  @doc """
+  Add this component to a `Scenic.Graph`
+  """
+  @callback add_to_graph(graph :: Scenic.Graph.t(), data :: any, opts :: list) :: Scenic.Graph.t()
+
+  @doc """
+  Verify that this the data for this component is valid.
+
+  Return an okay tuple if the data is valid and any other term if the data is
+  not valid. Here is an example implementation that checks if the input is a
+  valid binary:
```

Comment:
`okay tuple` stuck out to me (and I haven't seen it written like that before) and the closest thing I could find was `Retrieve an item from the cache and wrap it in an `{:ok, _}` tuple.` in the codebase. Maybe it should be written like that for consistency?

Reply:
I agree with @nyaray. Lets change the `okay tuple` part of the docs to `{:ok, data}` and then we are good.

Thank you for the docs!

Reply:
Yeah makes sense. Updated!